<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consultar</class>
 <widget class="QDialog" name="consultar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="idcliente">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="c_cliente">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Validar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="apellido">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Identificacion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Edad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_cuenta">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Validar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saldo">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="estado">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cuenta">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cuenta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idd">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Id Dueño</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre dueño</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="d_du">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
